### PR TITLE
FAQ: rewrite question about Windows after WSL support

### PR DESCRIPTION
**Pull request by Asartea**
## Because
- We support WSL2 now

## This PR
- Rewrites the featured question to mention using WSL

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
 [How to Ask Technical Questions](https://www.theodinproject.com/guides/community/how_to_ask)
 
-### ==== FEATURED QUESTION ====
+### ==== FEATURED QUESTION: Why can't I use Windows? ====
 
-[Why can't I use Windows?](https://discord.com/channels/505093832157691914/690588860085960734/828714982253789215)
+There is a whole [list of reasons](https://discord.com/channels/505093832157691914/690588860085960734/828714982253789215) for why we don't recommend using Windows.
 
-### =========================
+You can however use Windows Subsystem for Linux (WSL2), which lets you run a full-fledged Linux enviroment from within an existing Windows installation.
```
Comment (xandora) [inline FAQ.md:17]:
```suggestion
You can however use Windows Subsystem for Linux (WSL2), which lets you run a full-fledged Linux environment from within an existing Windows installation.
```

**Review by xandora — APPROVED** [review]
Looks good to me.

**Comment by Asartea** [comment]
@xandora is there a reason this hasn't been merged?

**Comment by xandora** [comment]
@Asartea I think I may have just forgotten.

**Comment by Asartea** [comment]
No problem, happens to the best of us

**Pull request merged**